src/ProjectTemplates/Web.ProjectTemplates/content/BlazorWeb-CSharp/BlazorWeb-CSharp/wwwroot/css/app.css

```diff
@@ -64,36 +67,3 @@ a, .btn-link {
     .blazor-error-boundary::after {
         content: "An error has occurred."
     }
-
-.loading-progress {
```

Comment:
I assume this means that the resolution to https://github.com/dotnet/aspnetcore/issues/49056 is that the loading progress UI just isn't used in this template because prerendering is enabled. Is that right?

Reply:
Not only because prerendering is enabled, but also because we would have nowhere sensible to show the loading UI. Any component might be a webassembly one, not just entire pages.